core/js/typespecs/app_types.ts

```diff
@@ -303,6 +303,15 @@ export const playgroundLogsLogIdLiteralSchema = z
   .literal("x")
   .describe("A literal 'x' which identifies the object as a playground log");
 
+// Defined as a separate openapi schema from promptDataSchema, so that the
+// nullish doesn't bleed into the openapi schema for the raw PromptData type.
+const promptDataNullishSchema = promptDataSchema
+  .nullish()
+  .openapi("PromptDataNullish");
+const functionTypeEnumNullish = functionTypeEnum
```

Comment:
#nit maybe worth sharing the schema

https://github.com/braintrustdata/braintrust-sdk/pull/853/files#diff-6b08eabf71a46b843175831585e3581b9516702233780ef00f5529cf79b4ba96R394

Reply:
not sure I follow?